components/Home/Home.vue

```diff
@@ -71,6 +73,66 @@ const device = useDevice()
 //
 // Hooks
 //
+onBeforeMount(async () => {
+  const favs = getHashPart(router, 'favs')
+  if (favs) {
+    try {
+      const addressHashes: string[] = []
+
+      favs
+        .split(',')
+        .forEach((id) => {
+          if (!Number.isNaN(Number(id)))
+            favoritesStore.toggleFavorite(Number(id))
+
+          if (id.startsWith('addr-'))
+            addressHashes.push(id.substring(5))
+        })
+
+      await Promise.all(addressHashes.map(async (hash) => {
+        const coordinates = decodeBase32(hash)
+        const address = await $fetch<ApiSearchResult<ApiAddrSearchResult>>(`${config!.API_ADDR}/reverse?lon=${coordinates.lng}&lat=${coordinates.lat}`)
+
+        if (!address || !address.features.length)
+          throw createError({ statusCode: 404, message: `Error while reverse geocoding: ${hash}` })
+
+        favoritesStore.toggleFavoriteAddr({
+          type: 'Feature',
+          geometry: address.features[0].geometry,
+          properties: {
+            internalType: 'address',
+            metadata: {
+              id: address.features[0].properties.id as ApiPoiId,
+            },
+            name: address.features[0].properties.label,
+            vido_zoom: address.features[0].properties.type === 'municipality'
+              ? MAP_ZOOM.selectionZoom.municipality
+              : MAP_ZOOM.selectionZoom.streetNumber,
```

Comment:
Were are not doing a geocoding. We should use a detailled zoom level in all case.

Reply:
Yep in favorite mode vido_zoom isn't even used I think.

[InitialBbox](https://github.com/teritorio/vido/blob/ebc5e01b71c0ab975b9415d4891882a4f6181cb7/components/Home/Home.vue#L289) is managing the zoom in this case :
```js
initialBbox.value = getBBoxFeatures(favoriteFeatures.value)
```